src/components/Dashboard/DashboardNextToJumpBanner.tsx

```diff
@@ -1,15 +1,21 @@
 import { useNextToJumpData } from "../../hooks/data";
 import { Loader } from "../Loader";
 import { DashboardBannerRow } from ".";
+import { useConfig } from "../../providers/Config";
 
 export const DashboardNextToJumpBanner: React.FC = () => {
   const { nextMeets } = useNextToJumpData();
+  const config = useConfig();
 
   return (
     <div className="flex lg:divide-x-2 divide-indigo-800 p-5 bg-indigo-600 rounded-lg mb-5 mt-1 text-white text-xs shadow-md overflow-hidden">
-      {nextMeets ? (
+      {nextMeets && config ? (
```

Comment:
you don't need to check for if `config` exists if it's an optional prop for `DashboardBannerRow`